pkgs/top-level/linux-kernels.nix

```diff
@@ -598,8 +598,6 @@ in
 
         netatop = callPackage ../os-specific/linux/netatop { };
 
-        prl-tools = callPackage ../os-specific/linux/prl-tools { };
```

Comment:
Could you add a `throw` for the package? Thanks a lot.

Reply:
Sure, added in latest iteration.